flink-core/src/main/java/org/apache/flink/api/common/io/DelimitedInputFormat.java

```diff
@@ -84,12 +84,11 @@
 	 */
 	private static int MAX_SAMPLE_LEN;
 
-	static { loadGlobalConfigParams(); }
-	
-	protected static void loadGlobalConfigParams() {
-		int maxSamples = GlobalConfiguration.getInteger(ConfigConstants.DELIMITED_FORMAT_MAX_LINE_SAMPLES_KEY,
+
+	protected static void loadConfigParameters(Configuration parameters) {
```

Comment:
According to the plugin which checks API changes, it is not breaking. I don't know whether this is a shortcoming of the plugin or actually ok. We can still keep the old method.